packages/gatsby-plugin-clerk/src/ssr/authenticateRequest.ts

```diff
@@ -17,7 +17,10 @@ export function authenticateRequest(context: GetServerDataProps, options: WithSe
     clientUat: cookies[constants.Cookies.ClientUat],
     host: headers.get(constants.Headers.Host) as string,
     forwardedPort: headers.get(constants.Headers.ForwardedPort) as string,
-    forwardedHost: returnReferrerAsXForwardedHostToFixLocalDevGatsbyProxy(headers),
+    forwardedHost:
+      process.env.NODE_ENV === 'development'
```

Comment:
Hey Stef, can you share some more insight about this change?

Reply:
@nikosdouvlis  Sure! `returnReferrerAsXForwardedHostToFixLocalDevGatsbyProxy` is used to resolve an issue in the local dev environment. I made sure it is used only in `development` environment because I spotted it had issues on production.